### PR TITLE
[DS-241]: [Pyxis React] Implementation of CheckboxGroup and RadioGroup components

### PR DESCRIPTION
**Pull request by lauratrapani**
https://prima-assicurazioni-spa.myjetbrains.com/youtrack/issue/DS-241

## 🚀 New Features
Implementation of RadioGroup and CheckboxGroup components in React

## 📝 Docs
The documentation of the RadioGroup and CheckboxGroup has been updated.
The CSS documentation is now available on Zeroheight:

- [RadioGroup](https://prima.zeroheight.com/styleguide/s/79372/p/4420b8-radio-button/b/785ec2)
- [CheckboxGroup](https://prima.design/3794e337c/p/45aac0-checkbox/t/28e189)

### CODE REVIEW

**Review by alessiosantangelo — APPROVED** [review]
LGTM

File: packages/pyxis-react/src/stories/components/Radio/Overview.tsx [hunk 1]
```diff
@@ -155,15 +104,15 @@ const Overview: FC = () => (
...
-    <OverviewTemplate description={classDescription} title="Overview of CSS classes">
-      <Table
-        body={tableClassBody}
-        gridTemplateColumns="300px 1fr 100px 2fr"
-        head={['Class', 'Type', 'Apply to', 'Note']}
-      />
+    <OverviewTemplate description={apiDescription} title="Component API">
+      <p className="title-s-bold c-neutral-base">RadioGroup</p>
+      <ArgsTable exclude={['type', 'hasLabel']} of={RadioGroup} />
+      <p className="title-s-bold c-neutral-base">Checkbox</p>
```
Comment (emanuelemacri) [inline packages/pyxis-react/src/stories/components/Radio/Overview.tsx:112]:
```suggestion
      <p className="title-s-bold c-neutral-base">Radio</p>
```

**Review by emanuelemacri — CHANGES_REQUESTED** [review]
un paio di cosette

File: packages/pyxis-react/src/components/Form/CheckboxGroup/CheckboxGroup.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React, { FC } from 'react';
+import ControlGroup, { ControlGroupProps } from 'internalComponents/ControlGroup';
+import Checkbox, { CheckboxProps } from './Checkbox';
+import { OptionLayout } from '../../../internalComponents/ControlGroup/ControlGroup';
```
Comment (emanuelemacri) [inline packages/pyxis-react/src/components/Form/CheckboxGroup/CheckboxGroup.tsx:4]:
```suggestion
import { OptionLayout } from 'internalComponents/ControlGroup/ControlGroup';
```

**Review by emanuelemacri — CHANGES_REQUESTED** [review]
ultimissima cosettina

**Pull request merged**